pkg/bosh/manifest/dns.go

```diff
@@ -207,12 +237,43 @@ func (dns *boshDomainNameService) Reconcile(ctx context.Context, namespace strin
 	return nil
 }
 
+func (dns *boshDomainNameService) createCorefile(namespace string) string {
+	rewrites := make([]string, 0)
+	for _, alias := range dns.Aliases {
+		for _, target := range alias.Targets {
+			from := alias.Domain
+			if target.Query == "_" {
+				from = strings.Replace(from, "_", target.InstanceGroup, 1)
+			}
+			to := fmt.Sprintf("%s.%s.svc.cluster.local", dns.HeadlessServiceName(target.InstanceGroup), namespace)
+			rewrites = append(rewrites, fmt.Sprintf("rewrite name exact %s %s", from, to))
+		}
+	}
+
+	tmpl := template.Must(template.New("Corefile").Parse(`
+.:8053 {
+    errors
+    health
+    {{- range $_, $rewrite := .Rewrites }}
```

Comment:
I thought (for `text/template` / `html/template`) that you don't need the `$_` and it assigns values (instead of indices) by default?

(We can do `range $rewrite := .` just fine too, I think, so we don't _need_ the temporary struct… but that probably doesn't hurt anyway)

Reply:
Fixed in https://github.com/cloudfoundry-incubator/cf-operator/pull/642/commits/5c99d7112830100b0fccf42bbb8943671bfe4fab.